src/components/Search/SearchForm.jsx

```diff
@@ -1,53 +1,84 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { browserHistory } from 'react-router';
+import { debounce } from 'lodash';
```

Comment:
If debounce isn't too hard to implement without using lodash, I'd like to at some point remove our dependency on lodash at some point to reduce bundle size.